### PR TITLE
Don't apply log_softmax since CrossEntropyLoss expects logits

### PR DESCRIPTION
**Pull request by balvisio**
Hello @mlabonne ,
Thanks for the quick reply. I found another minor issue. According to the Pytorch documentation `CrossEntropyLoss` expects the logits so the `forward()` method shouldn't apply `log_softmax()`.

Thanks!

### CODE REVIEW

**Comment by balvisio** [comment]
Hi @mlabonne : 
I just added another small fix in a new commit. I noticed that in the fit() function when using the `NeighborLoader` the `total_loss` was being divided by `len(loader.dataset)` which in the case of the `NeighborLoader` returns the total number of nodes since it is an instance of a `NodeLoader`. IIUC we really want to divide by the total number of graphs so it is more appropriate to divide by `len(loader.data)`. (Dividing by `len(loader.dataset)` is appropriate when a `DataLoader` was used)

Thanks!
